src/api/app/components/reports_notice_component.html.haml

```diff
@@ -0,0 +1,6 @@
+.text-warning
+  %i.fas.fa-flag
+  - if by_user
+    You reported this #{reportable.class.name.downcase}.
+  - else
+    This #{reportable.class.name.downcase} has #{pluralize(reports.count, 'report')}.
```

Comment:
This `pluralize(...)` can go inside the `reports` method in the component class. WDYT?

Reply:
What about making `reports` method an instance variable then?